third_party/move/move-compiler-v2/tests/checking-lang-v1/conditional_global_operations.exp

```diff
@@ -1,25 +1,25 @@
 
 Diagnostics:
-error: access specifier not enabled. Only plain `acquires <ResourceName>` is enabled.
-   ┌─ tests/acquires-checker/conditional_global_operations.move:14:49
+error: access specifiers not supported in language versions before `2.0-unstable`
```

Comment:
As given, this error message is bound to confuse the reader. It would seem that `acquires` is indeed supported in older versions of the language, it is just that instantiated types are not allowed. Maybe there is already an issue to tackle this, if not, it would be good to file one.

Reply:
Improved the messages